apps/studio/pages/project/[ref]/api/graphiql.tsx

```diff
@@ -28,35 +31,54 @@ const GraphiQLPage: NextPageWithLayout = () => {
   const { data: settings, isFetched } = useProjectApiQuery({ projectRef })
 
   const apiService = settings?.autoApiService
-  const anonKey = apiService?.service_api_keys.find((x) => x.name === 'anon key')
-    ? apiService.defaultApiKey
+  const serviceRoleKey = apiService?.service_api_keys.find((x) => x.name === 'service_role key')
+    ? apiService.serviceApiKey
     : undefined
 
+  const { data: config } = useProjectPostgrestConfigQuery({ projectRef })
+  const jwtSecret = config?.jwt_secret
+
   useEffect(() => {
     if (ui.selectedProjectRef) {
       // Schemas may be needed when enabling the GraphQL extension
       meta.extensions.load()
     }
   }, [ui.selectedProjectRef])
 
-  const graphqlUrl = `${API_URL}/projects/${projectRef}/api/graphql`
-
   const fetcher = useMemo(() => {
-    const fetcherFn = createGraphiQLFetcher({ url: graphqlUrl, fetch })
+    const fetcherFn = createGraphiQLFetcher({
+      url: `${API_URL}/projects/${projectRef}/api/graphql`,
```

Comment:
just FYI ideally this endpoint does the key retrieval, rather than FE passing it over
not related to this PR but the graphiql interface wouldn't work for a read-only org member since they dont have access to API keys